lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
@davidtheclark your code is working, but it breaks `system-tests/fix/fix.test.js`, because this test is using other `os` method. I tried to find a way to mock only one method for `os`. But no luck here :(